pkg/controller/pv-monitor-controller/pv_monitor_controller.go

```diff
@@ -84,15 +89,16 @@ func NewPVMonitorController(client kubernetes.Interface, monitorName string, con
 	eventRecorder = broadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: fmt.Sprintf("csi-pv-monitor-controller-%s", monitorName)})
 
 	ctrl := &PVMonitorController{
-		csiConn:         conn,
-		eventRecorder:   eventRecorder,
-		client:          client,
-		monitorName:     monitorName,
-		monitorInterval: monitorInterval,
-		pvQueue:         workqueue.NewNamed("csi-monitor-pv-queue"),
+		csiConn:            conn,
+		eventRecorder:      eventRecorder,
+		supportListVolumes: supportListVolumes,
+		enableNodeWatcher:  enableNodeWatcher,
+		client:             client,
+		monitorName:        monitorName,
+		monitorInterval:    monitorInterval,
+		pvQueue:            workqueue.NewNamed("csi-monitor-pv-queue"),
 		//pvQueue:       workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "csi-monitor-pv-queue"),
```

Comment:
Remove commented out line.

Reply:
Any reason not to use RateLimitingQueue?

Reply:
We check PVs health condition periodically, do not need rate limiting